.circleci/config.yml

```diff
@@ -62,7 +66,12 @@ jobs:
           command: mix format --check-formatted
       - run:
           name: "Run dialyzer"
-          command: mix dialyzer --plt
+          command: mix dialyzer
+      - save_cache:
+          key: v1-plt-cache-{{ checksum "mix.lock" }}
+          paths:
+            - '_build'
+            - '~/.mix'
```

Comment:
What do you think about adding `~/.mix` to the `elixir-build-...` key instead of introducing a new cache key, since we are already caching `_build` there? We could also probably change the that key to rely on `mix.lock` checksum instead of `mix.exs`.

Reply:
We can't add it to the `elixir-build` key because we can't cache that directory until after the `mix dialyzer` task is run, since that's what generates the PLTs. We could cut the `elixir-build` step all together and then do the build & PLT generation in the `test` task, and then cache both directories at that point.

I'll put up another commit with what that would look like, and then you can pick which one you like better.